test/specs/configs/full.json

```diff
@@ -240,7 +246,12 @@
         "location": ""
       },
       "sticky": false,
-
+      "redirects": [
```

Comment:
Testing the "multiple" case (more than one redirect defined) would be great too.